### PR TITLE
Add e2e tests and docs for v0.56.15

### LINKED ISSUE #1881: Notice of K8s Release
A Kubernetes release was detected as having occurred in the last 24h. The ./scripts/gather_e2e_data.sh script should be run and Sonobuoy tested against the new release.

### PR DESCRIPTION
**Pull request by joyvuu-dave**
Signed-off-by: Dave Riddle <driddle@vmware.com>

**What this PR does / why we need it**:
Add e2e tests and docs for v0.56.15

**Which issue(s) this PR fixes**
- Fixes #1881, #1882, #1883

### CODE REVIEW

**Comment by pydctw** [comment]
Thanks for updating the e2e data from the new k8s releases, @joyvuu-dave!

File: .github/ISSUE_TEMPLATE/weekly-release.md [hunk 1]
```diff
@@ -43,14 +51,14 @@ version of the docs. The `-b` flag also bumps the version in the code to match.
 
 ```
 git tag -a v0.x.y -m "Release v0.x.y"
-git push upstream refs/tags/v0.x.y
+git push origin refs/tags/v0.x.y
```
Comment (pydctw) [inline .github/ISSUE_TEMPLATE/weekly-release.md:54]: I think we need to leave it at `upstream`. Is your setting similar to below?

```
➜  sonobuoy git:(main) git remote -v
origin	git@github.com:YOUR_GITHUB_ID/sonobuoy.git (fetch)
origin	git@github.com:YOUR_GITHUB_ID/sonobuoy.git (push)
upstream	git@github.com:vmware-tanzu/sonobuoy.git (fetch)
upstream	no_push (push)
```

File: .github/ISSUE_TEMPLATE/weekly-release.md [hunk 2]
```diff
@@ -43,14 +51,14 @@ version of the docs. The `-b` flag also bumps the version in the code to match.
...
 ```
 
 If there is a problem and you need to remove the tag, run the following commands
 
  ```
  git tag -d v0.x.y
- git push upstream :refs/tags/v0.x.y
+ git push origin :refs/tags/v0.x.y
```
Comment (pydctw) [inline .github/ISSUE_TEMPLATE/weekly-release.md:61]: same as above

File: .github/ISSUE_TEMPLATE/weekly-release.md (hunk 1 above)
Comment (joyvuu-dave) [reply .github/ISSUE_TEMPLATE/weekly-release.md:54]: I believe `origin` is the default when doing a git clone and that's how mine shows up. https://git-scm.com/docs/git-clone

```
➜  sonobuoy git:(main) git remote -v
origin	git@github.com:vmware-tanzu/sonobuoy.git (fetch)
origin	git@github.com:vmware-tanzu/sonobuoy.git (push)
➜  sonobuoy git:(main)
```

Comment (pydctw) [reply .github/ISSUE_TEMPLATE/weekly-release.md:54]: I understand where you are coming from. 

The convention in OSS community (at least in k8s community) is to fork the repo, set upstream for the OSS repo and set origin as your forked repo. That is why the instruction says to push to `upstream`.

Comment (pydctw) [reply .github/ISSUE_TEMPLATE/weekly-release.md:54]: See [k8s github-workflow](https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md?plain=1#L45-L62) for more details

Comment (joyvuu-dave) [reply .github/ISSUE_TEMPLATE/weekly-release.md]: Ah, I see! I've swapped it back.

**Comment by joyvuu-dave** [comment]
@pydctw Do you know what we need to do to make this PR mergable? @JamesBurchell tried to see if he could add an approval but didn't see anything.

**Pull request merged**